### DISCUSSION TITLE
No Fire 🔥🔥🔥🔥

### DISCUSSION THREAD

**Discussion by N4TUR4L20** [firemodels/fds#12960] [Q&A]
I have run the simulation, but no fire appears, what is wrong?

please help and thank you

Here is the code :
[methane.txt](https://github.com/firemodels/fds/files/15368093/methane.txt)

**Comment by TristanHehnen** [comment]
The file is empty.

**Reply by N4TUR4L20** [reply]
thank you, the file has been updated

**Comment by TristanHehnen** [comment]
Please elaborate a bit on what you would like to achieve.

**Reply by N4TUR4L20** [reply]
I want to make a pool fire filled with methanol fuel and when the room temperature reaches a certain limit, the sprinkle turns on and extinguishes the fire. I want to take data in the form of extinguishing time, temperature and oxygen levels at each time.

**Reply by TristanHehnen** [reply]
Are you interested in the gas phase combustion being suppressed, or do you want to look at the interaction of the flame heat feedback evaporating the methanol? 

I gather that you looked at the FDS examples and specifically at `methanol_evaporation.fds`. 
- When you compare both setups in Smokeview (`T_END = 0`), is the pool defined as you intended to (look also in the user guide for `SURF_IDS` in section 7.2.1)?
- In `methanol_evaporation.fds` the parameter `EXTERNAL_FLUX` is used. Have a look in the user guide what it does.
- You added `RAMP_Q` to your `SURF ID='POOL'`. What does this do?

**Reply by drjfloyd** [reply]
Not why you aren't seeing a fire but a couple of other comments:

This input says that at t=0, the volume defined by XB is over 50 % liquid water. Is this what you intend?  

&INIT PART_ID='water_drops', N_PARTICLES=1, MASS_PER_VOLUME=523.6, XB=0,0.5,0.5,0,0,1.1/

In your two part inputs you have used the same ID twice. If you want two different particles use two different IDs. Also, make sure you are using the correct units. DIAMETER=100000 is a 10 cm diameter droplet which is not someting that would be physically meaningful in FDS.

&PART ID='water_drops', SPEC_ID='WATER VAPOR', SAMPLING_FACTOR=1, QUANTITIES='PARTICLE DIAMETER', DIAMETER=2000./
&PART ID='water_drops', SPEC_ID='WATER VAPOR', STATIC=.TRUE., QUANTITIES(1:2)='PARTICLE DIAMETER','PARTICLE TEMPERATURE',
      DIAMETER=100000., INITIAL_TEMPERATURE=20., SAMPLING_FACTOR=1, MONODISPERSE=.TRUE./


Also, consider what happens when you add water to methanol. They are soluable in each other. This is a process that FDS does not model. In fact FDS is going to treat the liquid pool as a solid surface.
